Comments.md

```diff
@@ -0,0 +1,148 @@
+# Comments
+
+In general, follow the comments section in the [official guidelines](https://swift.org/documentation/api-design-guidelines/#fundamentals).
```

Comment:
I'm curious - is there anything below where we differ from the official guidelines or adding something on top of it. It kinda feels like we are just restating it so a thought might be to explicitly mention how each guideline below diverges from official guidelines?

Reply:
The official guidelines only covers documentation and not inline comments. In addition, our guideline makes having documentation mandatory for all publicly facing functions.

Reply:
Yeah, there's unfortunately very little about comments in the official guidelines. And as mentioned, it doesn't cover anything about inline comments or styles of writing.